firebase-messaging-sw.tmpl

```diff
@@ -18,16 +18,10 @@
 importScripts('https://www.gstatic.com/firebasejs/3.5.2/firebase-app.js');
 importScripts('https://www.gstatic.com/firebasejs/3.5.2/firebase-messaging.js');
 
-const configUrl = '/messaging-config.json';
-caches.match(configUrl)
-  .then(response => response || fetch(configUrl))
-  .then(response => response.json())
-  .then(config => {
-    firebase.initializeApp({
-      messagingSenderId: config.firebase_msg_sender_id
-    });
-    const messaging = firebase.messaging();
-    messaging.setBackgroundMessageHandler(_ => {
-      return self.registration.showNotification();
-    });
-  });
+firebase.initializeApp({
+  messagingSenderId: '<%= firebaseMsgSenderId %>'
```

Comment:
Might be better to have the injected config in one place (just this one key at the moment), and reference it here via something like `__config.firebaseMsgSenderId`. (Similar to the way we reference `__config` from the page JS.) Not that bothered though--could wait for another key to be added. Generating code is all kinds of icky…